opts/opts.go

```diff
@@ -203,6 +203,9 @@ func ValidatePath(val string) (string, error) {
 	}
 
 	splited := strings.SplitN(val, ":", 2)
+	if splited[0] == "" {
+		return val, fmt.Errorf("bad format for volumes: %s", val)
+	}
 	if len(splited) == 1 {
 		containerPath = splited[0]
 		val = path.Clean(splited[0])
```

Comment:
I'm also wondering if we should modify this in this PR or on another one, just for that (so it's easier to review the change and easier to revert too). @duglin WDYT ?